source/user-manual/wazuh-dashboard/config-file.rst

```diff
@@ -724,6 +724,105 @@ Set the footer of the PDF reports. To use an empty footer, type a space " " in t
+
++--------------------+---------------------------------------------------------------------+
+| **Default value**  | ["wazuh-alerts\*", "wazuh-archives\*", "-wazuh-alerts-4.x-sample*"] |
++--------------------+---------------------------------------------------------------------+
+| **Allowed values** | Array of strings. Eg: ["wazuh-archives-\*"]                         |
++--------------------+---------------------------------------------------------------------+
+
+ism.rollover.priority
+^^^^^^^^^^^^^^^^^^^^^
+
+The priority value for the policy. The higher the value, the higher the priority. If there are multiple policies to manage an index, Wazuh applies the one with the highest priority value. There cannot be two policies with the same priority value to manage the same index.
+
++--------------------+-----------------------------------+
+| **Default value**  | 50                                |
++--------------------+-----------------------------------+
+| **Allowed values** | Any integer number greater than 0 |
++--------------------+-----------------------------------+
+
+ism.rollover.min_index_age
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The minimum age required to roll over the index. Index age is the time elapsed since its creation. Supported units are ``d`` (days), ``h`` (hours), ``m`` (minutes), ``s`` (seconds), ``ms`` (milliseconds), and ``micros`` (microseconds).
+
+Rolling over an index too often might cause performance issues. The minimum recommended value is 7 days.
+
++--------------------+---------+
+| **Default value**  | 7d      |
++--------------------+---------+
+| **Allowed values** | String  |
++--------------------+---------+
+
+ism.rollover.min_primary_shard_size
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The minimum storage size in GB of a single primary shard required to roll over the index. For example, if you set ``min_primary_shard_size`` to 30 GB and one of the primary shards in the index has a size greater than the condition, the rollover occurs.
+
+The recommended shard sizes are 10-30 GB for search-heavy workloads and 30-50 GB for write-heavy workloads. The shard size must never exceed the 50 GB. In Wazuh, we use 25 GB, in order to achieve a good balance between both workloads.
+
++--------------------+-----------------------------------+
+| **Default value**  | 25                                |
++--------------------+-----------------------------------+
+| **Allowed values** | Any positive number               |
++--------------------+-----------------------------------+
+
+ism.rollover.min_doc_count
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The minimum number of documents required to roll over the index.
+
+The documents are distributed across the shards of the index. Although a shard can contain up to 2^31 documents, it's recommended to keep the documents-per-shard below 200 million.
```

Comment:
I understand it's the total number of documents and not a count number per shard.